### ISSUE TITLE
Dark theme is broken in additional inbox sections

### ISSUE THREAD

**Issue by villelahdenvuo** [leggett/simplify#16]
As seen in the screenshot, additional inbox sections don't work correctly in dark more:

![image](https://user-images.githubusercontent.com/633524/55635813-e42d7600-57c1-11e9-8234-c6ca1ed94823.png)

**Comment by kennydorman** [comment]
It actually looks to be an issue with read email, I have the same issue with the tabbed inbox

**Comment by kennydorman** [comment]
Quick fix for anyone with this issue, you can create a bookmarklet with the following URI or simply "Go" to this URL (copy/paste to url bar) 
```
javascript:(function(){var css = document.createElement("style");css.innerHTML = ".yO {background: rgba(0,0,0,0.8) !important;color: rgba(255,255,255,0.70) !important;}";document.body.appendChild(css);})()
```

You'll have to do this for every new gmail session

**Comment by leggett** [comment]
Correct. Known issue and on the list. 

I set the background for read items to be white with a lighter font so most themes don't have the zebra stripes but that manual set breaks dark themes. I'm either going to find a way to detect the background (document.getElementsByClassName('yO')[0].style.background returns null) or I'll abandon dropping the read-state background color. I also tried just using a filter:contrast(120%) but that messes with the borders.
